src/main/java/com/alexx/employeesAndDepartments/controllers/EmployeeController.java

```diff
@@ -26,13 +28,17 @@ public EmployeeController(EmployeeService employeeService) {
     @Operation(summary = "Add employee", description = "Add new employee, Request body with name of employee required, " +
             "ID is generated automatically, birthDate must be over 18 years ago, idOfDepartment must match the existing department")
     public ResponseEntity<EmployeeDTO> addEmployee(@RequestBody EmployeeDTO employeeDTO) {
+        log.info("Add employee...");
```

Comment:
Please refer me a link where you read that you should do logging like this. And please describe in this comment best logging practices that you learnt

Reply:
My logic is as follows:
first of all in INFO I write the essence of the called method (e.g. Add department),
then in DEBUG I write key parameters to be checked (name, ID, etc.),
if request completed, I write in INFO about it (e.g. New employee saved with ID = 1.)
if an exception has occurred, I write in WARN: (Name of employee must not be empty) or (Employee with this ID does not exist)

Reply:
Okay, sounds good. The only thing could you please remove dots from the end e.g. add employee...